config/samples/onload/onload-module/mkdist-direct/Dockerfile

```diff
@@ -16,14 +16,15 @@ RUN mkdir -p /build/onload
 RUN tar xzf onload.tar.gz -C /build/onload --strip-components=1
 WORKDIR /build/onload/
 
-ENV i_prefix=/opt
-RUN scripts/onload_build --kernel --kernelver $KERNEL_FULL_VERSION $ONLOAD_BUILD_PARAMS && \
-    scripts/onload_install --nobuild --kernelfiles --kernelver $KERNEL_FULL_VERSION && \
-    make -j8 -C src/driver/linux_net CONFIG_SFC_VDPA= CONFIG_SFC_MTD= KVER=$KERNEL_FULL_VERSION && \
-    /sbin/depmod -b /opt $KERNEL_FULL_VERSION
```

Comment:
Presumably we don't need the explicit sfc build? Was this previously a workaround?